### DISCUSSION TITLE
How can I run TextPrediction without GPU?

### DISCUSSION THREAD

**Discussion by icarosun** [autogluon/autogluon#1595] [Q&A] [answered]
Is it possible?

**Comment by Innixma** [comment, marked as answer]
You can run TextPredictor without GPU starting in v0.4.0 release (Releasing on March 9th). Do note that training may be very slow, so an alternative is to pass the same data into TabularPredictor, which has strong text support and can train quickly on CPU.

**Reply by icarosun** [reply]
I tried to run the TextPrediction - QuickStart in Colab, but this error occurred:

![image](https://user-images.githubusercontent.com/46682140/157151784-e5328f19-486c-46e7-afd7-a45e3e882724.png)

So, to execute in that version v0.4.0, I just to import like this:

`import "autogluon<0.4.0`

Thanke you!

**Reply by Innixma** [reply]
AutoGluon v0.4 is now released, you should be able to simply do `pip install autogluon` with a fresh virtual environment.

**Reply by Innixma** [reply]
I have confirmed that AutoGluon v0.4 TextPredictor works on CPU: https://colab.research.google.com/drive/1lYhCkJq8ZAj8OiH5rEjSbCVLDiVREXVl?usp=sharing

I wouldn't recommend it though. Training is very slow and if you don't have a GPU, consider using TabularPredictor instead which is much faster.
